Makefile

```diff
@@ -0,0 +1,54 @@
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+PROJECT_ROOT := $(dir $(MKFILE_PATH))
+TRANSCENDENCE_DEPS := $(shell find ./transcendence-app/ -type f -name "*.ts" \
+                              | grep -v "node_module\|dist\|test" \
+                              | grep -i "controller\|dto\|domain\|entity")
+DOCKER_COMPOSE := $(shell $(PROJECT_ROOT)/scripts/get-docker-compose.sh)
+ifeq ($(SEED),)
+SEED := seed
+endif
+
+.PHONY: all
+all: gen
+	$(DOCKER_COMPOSE) up --build -d
+
+transcendence-app/swagger-spec.yaml: $(TRANSCENDENCE_DEPS)
+	$(PROJECT_ROOT)/scripts/generate-openapi.sh
+
+.PHONY: gen
+gen:
+	make transcendence-app/swagger-spec.yaml
+
+.PHONY: clean
+clean:
+	rm -rf $(PROJECT_ROOT)/webapp/src/shared/generated || true
+	rm $(PROJECT_ROOT)/transcendence-app/swagger-spec.yaml || true
+	$(DOCKER_COMPOSE) down -v
+
+.PHONY: re
+re:
+	make clean
+	make
+
+.PHONY: log-tr
+log-tr:
+	$(DOCKER_COMPOSE) logs -f transcendence-app
+
+.PHONY: log-wa
+log-wa:
+	$(DOCKER_COMPOSE) logs -f webapp
+
+.PHONY: log-db
+log-db:
+	$(DOCKER_COMPOSE) logs -f db
+
+.PHONY: get-ip
+get-ip:
+	@docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ft_transcendence-webapp-1
+
```

Comment:
I don't know why, or how can I change it the name, but in my computer ft_transcendence-webapp-1 is ft_transcendence_webapp_1 so this rule doesn't work unless I manually modify the makefile. Any idea how can I fix this?
And I sometimes use the ip of the db to connect to db manager so would be nice to have something like this: 
```
@echo "webapp ip:"
@docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ft_transcendence_webapp_1
@echo "transcendence-app ip:"
@docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ft_transcendence_transcendence-app_1
@echo "db ip:"
@docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ft_transcendence_db_1
```

Reply:
this should be fixed now, and be agnostic to your container name flavor